applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c

```diff
@@ -1309,71 +1205,18 @@ int unicast_client_stop(void)
 int unicast_client_send(struct le_audio_encoded_audio enc_audio)
 {
 	int ret;
-	size_t data_size_pr_stream;
-	struct bt_iso_tx_info tx_info = {0};
-	struct sdu_ref_msg msg;
-
-	if ((enc_audio.num_ch == 1) || (enc_audio.num_ch == ARRAY_SIZE(headsets))) {
-		data_size_pr_stream = enc_audio.size / enc_audio.num_ch;
-	} else {
-		LOG_ERR("Num encoded channels must be 1 or equal to num headsets");
-		return -EINVAL;
-	}
+#if CONFIG_BT_AUDIO_TX
+	static struct bt_bap_stream *bap_tx_streams[CONFIG_BT_BAP_UNICAST_CLIENT_ASE_SNK_COUNT];
```

Comment:
It is static, so it is only created once. However, I do see your point on the initialization. This was to handle dynamic changes. This may not be needed. Discuss.